org.eclipse.winery.frontends/app/topologymodeler/src/app/node/node.component.html

```diff
@@ -30,12 +30,13 @@
         {{ nodeTemplate.state }}
     </div>
 
-    <div *ngIf="policyIcons">
+    <div *ngIf="policyIcons" style="position: absolute; top: -27px">
```

Comment:
Avoid using inline CSS

Reply:
I tried to use a CSS class, but somehow it did not get applied even though it was shown in the inspector...

Reply:
Maybe try a more specific selector?!